packages/versioning/test/versioning.test.ts

```diff
@@ -1037,13 +1037,7 @@ describe("versioning: logic", () => {
         `
       )) as { MyService: Namespace };
 
-      const [v1, v2] = runProjections(runner.program, MyService);
-      const w1 = v1.projectedTypes.get(MyService) as Namespace;
-      const w2 = v2.projectedTypes.get(MyService) as Namespace;
-      w1.models.get("Widget")?.properties.size === 1;
-      w1.operations.get("create")?.parameters.properties.size === 1;
-      w2.models.get("Widget")?.properties.size === 2;
-      w2.operations.get("create")?.parameters.properties.size === 2;
```

Comment:
Why were these removed?

Reply:
those weren't actually running assertion and tbh after looking at it not sure they had much more value so I just kept the current state of just running the projection and making sure it doesn't carash